docker-compose.yml

```diff
@@ -1,14 +1,16 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright (c) 2022 Dell Inc, or its subsidiaries.
 ---
-version: "3.7"
+# version: "3.7" - version is obsolete and can be removed/commented out
```

Comment:
If we don't need it, then we can just remove it.
However, are we sure that it won't break anything if an older version of docker compose executable is used?

Reply:
It shouldn't break anything.  The version usage was merged into the docker-compose back in October 2022 and deprecated (but still allowed).  At this time the version could be removed from the files and they would still work properly.  In June of 2023, the removal was put into effect in the documentation and the linting checks followed after that.

In general we need to clean-up the docker-compose files for this across the repos.  I am also looking at PRs for this across the various repo's